src/core.js

```diff
@@ -1,4 +1,4 @@
-import { window, setTimeout, console } from "./globals";
+import { window, setTimeout, console, sessionStorage } from "./globals";
```

Comment:
This is well on its way, but I'd still like to see global `sessionStorage` become e.g. `QUnit.config.storage`. Do you think that should instead be a followup PR?